{
	"cve": {
		"CVE_data_meta": {
			"ASSIGNER": "psirt@us.ibm.com",
			"ID": "CVE-2022-30614"
		},
		"data_format": "MITRE",
		"data_type": "CVE",
		"data_version": "4.0",
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "IBM Cognos Analytics 11.1.7, 11.2.0, and 11.2.1 is vulnerable to a denial of service via email flooding caused by sending a specially-crafted request. A remote attacker could exploit this vulnerability to cause the server to consume all available CPU resources. IBM X-Force ID: 227591."
				}
			]
		},
		"problemtype": {
			"problemtype_data": [
				{
					"description": [
						{
							"lang": "en",
							"value": "NVD-CWE-noinfo"
						}
					]
				}
			]
		},
		"references": {
			"reference_data": [
				{
					"name": "ibm-cognos-cve202230614-dos (227591)",
					"tags": [
						"VDB Entry",
						"Vendor Advisory"
					],
					"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/227591"
				},
				{
					"name": "ibm-cognos-cve202230614-dos (227591)",
					"tags": [
						"VDB Entry",
						"Vendor Advisory"
					],
					"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/227591"
				},
				{
					"name": "https://security.netapp.com/advisory/ntap-20221014-0005/",
					"tags": [
						"Third Party Advisory"
					],
					"url": "https://security.netapp.com/advisory/ntap-20221014-0005/"
				},
				{
					"name": "https://security.netapp.com/advisory/ntap-20221014-0005/",
					"tags": [
						"Third Party Advisory"
					],
					"url": "https://security.netapp.com/advisory/ntap-20221014-0005/"
				},
				{
					"name": "https://www.ibm.com/support/pages/node/6615285",
					"tags": [
						"Patch",
						"Vendor Advisory"
					],
					"url": "https://www.ibm.com/support/pages/node/6615285"
				},
				{
					"name": "https://www.ibm.com/support/pages/node/6615285",
					"tags": [
						"Patch",
						"Vendor Advisory"
					],
					"url": "https://www.ibm.com/support/pages/node/6615285"
				}
			]
		}
	},
	"impact": {
		"baseMetricV3": {
			"cvssV3": {
				"attackComplexity": "LOW",
				"attackVector": "NETWORK",
				"availabilityImpact": "HIGH",
				"baseScore": 7.5,
				"baseSeverity": "HIGH",
				"confidentialityImpact": "NONE",
				"integrityImpact": "NONE",
				"privilegesRequired": "NONE",
				"scope": "UNCHANGED",
				"userInteraction": "NONE",
				"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
				"version": "3.1"
			},
			"exploitabilityScore": 3.9,
			"impactScore": 3.6
		}
	},
	"configurations": {
		"CVE_data_version": "4.0",
		"nodes": [
			{
				"cpe_match": [
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:11.1.7:-:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:*:*:*:*:*:*:*:*",
						"versionEndExcluding": "11.1.7",
						"versionStartIncluding": "11.1.0",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack1:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack2:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack3:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack4:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:*:*:*:*:*:*:*:*",
						"versionEndExcluding": "11.2.3",
						"versionStartIncluding": "11.2.0",
						"vulnerable": true
					}
				],
				"operator": "OR"
			},
			{
				"cpe_match": [
					{
						"cpe23Uri": "cpe:2.3:a:netapp:oncommand_insight:-:*:*:*:*:*:*:*",
						"vulnerable": true
					}
				],
				"operator": "OR"
			}
		]
	},
	"lastModifiedDate": "2024-11-21T07:03Z",
	"publishedDate": "2022-09-01T19:15Z"
}
